### ISSUE TITLE
Ho to make green default in pycharm or vscode ?

### ISSUE THREAD

**Issue by cocoonkid** [CleanCut/green#211]
Ho to make green default in pycharm or vscode ?

**Comment by cocoonkid** [comment]
I was able to do it via Pycharm Run External Tool Macro. But is there a more elegant solution?

**Comment by CleanCut** [comment]
@cocoonkid I don't use either of those myself, but I looked into it and here is what I've found:

For VSCode there are probably many options, but I suggest contacting the authors of the [Python extension](https://github.com/Microsoft/vscode-python) and asking them how to use Green via their extension.  I'm happy to work with them if they are interested (feel free to @-mention @cleancut in the issue, if you write one).

Similarly for Pycharm, I suggest contacting JetBrains, the company which makes Pycharm.  They [include support for other test runners](https://www.jetbrains.com/help/pycharm/pytest.html) so it may be possible to switch to a custom runner such as Green. If not, then you'll likely need to request support for it directly with them.  Again, I'm happy to work with them if they're interested in adding support.

I'm going to mark this closed as there are no action items that I need to follow up on, but feel free to keep on replying here if you would like to continue the discussion.
